release-notes/General/General_Feature_Release_10.5/General_Feature_Release_10.5.1.md

```diff
@@ -99,6 +99,95 @@ From now on, users will be allowed to give feedback (positive or negative) on al
+
+- As a DataMiner System Admin, you can easily extend your system with an extra node and move functionalities from existing nodes to new nodes, so you can rebalance your cluster.
+
+- Swarming makes it possible to recover functionalities from failing nodes by moving activities hosted on such a node to the remaining nodes.
+
+In a later iteration, the Swarming feature will also be able to assist in making rolling DataMiner software updates on live clusters possible, with limited downtime of specific functionality.
+
+> [!NOTE]
+> The above capabilities are possible with limited downtime and as long as there is spare capacity.
+
+##### Limitations
+
+Some functionality is currently not yet supported with the Swarming feature. These are the most important limitations at the moment:
+
+- Swarming is not available in DataMiner Systems with a [storage per DMA setup](xref:Configuring_storage_per_DMA) (Cassandra or MySQL).
+
+- Swarming is not available in DataMiner Systems with [Failover](xref:About_DMA_Failover).
+
+- Swarming is not available in DataMiner Systems where the [*LegacyReportsAndDashboards* soft-launch option](xref:Overview_of_Soft_Launch_Options#legacyreportsanddashboards) is enabled.
+
+- Swarming is currently limited to [basic elements](xref:SwarmingElements).
+
+> [!NOTE]
+> [Prerequisite checks](xref:EnableSwarming#running-a-prerequisites-check) are in place to prevent the enabling of the Swarming feature when non-supported objects are present. Where possible, you will also be prevented from configuring or creating these on a Swarming-enabled system.
+
+##### Required user permissions
+
+To enable the Swarming feature, the [Admin Tools](xref:DataMiner_user_permissions#modules--system-configuration--tools--admin-tools) user permission is required.
+
+Once the feature has been enabled, users will need the [Swarming](xref:DataMiner_user_permissions#modules--swarming) user permission to trigger any swarming actions. Users that previously had the [Import DELT](xref:DataMiner_user_permissions#general--elements--import-delt) and [Export DELT](xref:DataMiner_user_permissions#general--elements--import-delt) user permissions before Swarming was enabled will automatically also get the *Swarming* user permission.
+
+To swarm an element, users will also need config rights on the element.
+
+##### Swarming elements
+
+When Swarming has been enabled, you can swarm elements **in DataMiner Cube** via *System Center* > *Agents* > *Status*. On that page, the *Swarming* button will be displayed instead of the *Migration* button. Clicking the button will open a window where you can select the elements you want to swarm and the destination DMA.
+
+Swarming elements is also possible **via Automation**. For this, you will need an element key, i.e. the DMA ID/element ID pair of the element you want to swarm, and the target Agent ID. You can specify these as fixed values or as input variables. For example:
```

Comment:
beware: for the example below you'll need to also define two input script parameters in your script